computer_setup.md

```diff
@@ -80,9 +80,7 @@ sudo dpkg --add-architecture i386 && sudo apt update && sudo apt install wine32
 
 # Set up the wine install. If you already have one, you probably want to add an export WINEPREFIX=<some other path> to set up a dedicated install
 export WINEARCH=win32
-winetricks winxp
-winetricks sound=alsa
-winetricks vb6run 
+winetricks -q win7 vb6run pdh_nt4 sound=alsa
```

Comment:
I tried this, it appears that pdh_nt4 doesn't exist for me on either Debian Bullseye or Ubuntu impish. However, "pdh" does. However, if I try to use that (on Ubuntu) it results in
```
wine: Unimplemented function ADVAPI32.dll.PerfEnumerateCounterSet called at address 7BC76128 (thread 0009), starting debugger...
```

Is the pdh_nt4 trick available somewhere else?

Reply:
Sorry this took me a while to get back to! I ended up being pretty busy this week.

Yes, that makes sense.  I think that the version of winetricks that's available from the Debian repo is pretty old and doesn't include the new `pdh_nt4` verb. One solution might be to run `winetricks --self-update` before running `winetricks -q win7 vb6run pdh_nt4 sound=alsa`. Another solution might be ask users to install winetricks manually from the winetricks github using
```
sudo apt-get install cabextract -y
wget -q https://raw.githubusercontent.com/Winetricks/winetricks/master/src/winetricks
sudo chmod +x winetricks
sudo mv winetricks /usr/local/bin
```

Reply:
Makes sense. But that's more effort than just copying the DLL as written. I think I'll stick with that - but if you want to update the PR to do the other changes, happy to accept. Otherwise, I will try to push that update myself.

Reply:
That makes sense too. No worries. I’ll close this PR for now and just stick with the copying over of the DLL